src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,92 @@
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  let totalPages = 0;
+
+  if (perPage !== 0) {
+    totalPages = Math.ceil(total / perPage);
+  }
+
+  const pagesList = Array.from({ length: totalPages }, (_, index) => index + 1);
+
+  const showPages = pagesList.map(page => (
+    <li
+      key={page}
+      className={cn('page-item', { active: currentPage === page })}
+    >
+      <a
+        data-cy="pageLink"
+        className="page-link"
+        href={`#${page}`}
```

Comment:
Using a hash (`#`) in the `href` attribute can lead to unwanted behavior in the URL and might not be accessible. It's better to prevent the default action in the `onClick` handler and avoid changing the URL.